stylesheets/components/block/_main.styl

```diff
@@ -48,6 +48,42 @@
     &--wbb
       border-bottom: $border-100 dotted $grey-200
 
+    // Makes a block that "breaks out" of b-c and goes edge-to-edge
+    // (up to the above max-width), replicating the padding inside.
+    // Good for when you want a block to have a background that goes
+    // edge-to-edge.
+    &--hedge {
+      margin-left: -12.5%;
+      margin-right: -12.5%;
+      padding-left: 12.5%;
+      padding-right: 12.5%;
+    }
+
+    // compensates to remove the bottom padding from a b-c parent.
+    &--bedge {
+      margin-bottom: "calc(-1 * %s)" % ($sizing-700)
+    }
+
+  // Flex fill
+  &-ff
```

Comment:
Should probably be a modifier, so -- instead of -

Reply:
This one is its own block-type element, the equivalent of a `b-c`.